sdk/eventhub/azure-eventhub-checkpointstoretable/tests/test_storage_table_partition_manager.py

```diff
@@ -1,8 +1,181 @@
+    return  {
+    'fully_qualified_namespace': fully_qualified_namespace,
+    'eventhub_name': eventhub_name,
+    'consumer_group': consumer_group,
+    'partition_id': str(partition_id),
+    'owner_id': owner_id,
+    'etag': etag,
+    'last_modified_time': last_modified_time
+            }
+
+def _claim_ownership_exception_test(storage_connection_str, table_name):
+    fully_qualified_namespace = 'test_namespace'
+    eventhub_name = 'eventhub'
+    consumer_group = '$default'
+    ownership_cnt = 8
+
+    checkpoint_store = TableCheckpointStore.from_connection_string(
+        storage_connection_str, table_name)
+    ownership_list = []
+    for i in range(ownership_cnt):
+            ownership = _create_ownership(fully_qualified_namespace, eventhub_name, consumer_group, str(i), 'owner_id', None, None)
+            ownership_list.append(ownership)
+    result_ownership_list = checkpoint_store.claim_ownership(ownership_list)
+    assert result_ownership_list[0]['owner_id']  == 'owner_id'
+    single_ownership = [result_ownership_list[0].copy()]
+    single_ownership[0]['owner_id'] = "Bill"
+    ownership_list = checkpoint_store.claim_ownership(single_ownership)
+    assert ownership_list[0]['owner_id'] == 'Bill'
+
+    single_ownership = [result_ownership_list[0].copy()]
+    single_ownership[0]['etag'] =  'W/"datetime\'2021-08-02T00%3A46%3A51.7645424Z\'"'
+    single_ownership[0]['owner_id'] = 'Jack'
+    single_ownership[0]['partition_id'] = '10'
+    result_ownership = checkpoint_store.claim_ownership(single_ownership)
+    list_ownership =  checkpoint_store.list_ownership(fully_qualified_namespace, eventhub_name, consumer_group)
+    assert result_ownership[0] in list_ownership
+
+    single_ownership = [result_ownership_list[0].copy()]
+    single_ownership[0]['etag'] =  'W/"datetime\'2021-08-02T00%3A46%3A51.7645424Z\'"'
+    with pytest.raises(OwnershipLostError) as e_info:
+            checkpoint_store.claim_ownership(single_ownership)
+
+def _claim_and_list_ownership(storage_connection_str, table_name):
+    fully_qualified_namespace = 'test_namespace'
+    eventhub_name = 'eventhub'
+    consumer_group = '$default'
+    ownership_cnt = 8
+
+    checkpoint_store = TableCheckpointStore.from_connection_string(
+        storage_connection_str, table_name)
+    ownership_list = checkpoint_store.list_ownership(
+            fully_qualified_namespace,
+            eventhub_name,
+            consumer_group)
+    assert len(ownership_list) == 0
+
+    ownership_list = []
+
+    for i in range(ownership_cnt):
+            ownership = {}
+            ownership['fully_qualified_namespace'] = fully_qualified_namespace
+            ownership['eventhub_name'] = eventhub_name
+            ownership['consumer_group'] = consumer_group
+            ownership['owner_id'] = 'owner_id'
+            ownership['partition_id'] = str(i)
+            ownership['etag'] = None
+            ownership['last_modified_time'] = None
```

Comment:
I think you can call the `_create_ownership` method here instead?

Reply:
indeed good catch